Test suite build failure with servant 0.9

As seen on the Stackage build server:

```
Preprocessing test suite 'spec' for servant-mock-0.8.1...
[1 of 2] Compiling Servant.MockSpec ( test/Servant/MockSpec.hs, dist/build/spec/spec-tmp/Servant/MockSpec.o )

test/Servant/MockSpec.hs:69:13: error:
    • No instance for (ToHttpApiData TestHeader)
        arising from a use of ‘toApp’
    • In the first argument of ‘with’, namely ‘(toApp withHeader)’
      In the expression: with (toApp withHeader)
      In a stmt of a 'do' block:
        with (toApp withHeader)
        $ do { it "serves arbitrary response bodies"
               $ do { get "/"
                      `shouldRespondWith`
                        200 {matchHeaders = return $ MatchHeader $ ...} } }
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,0 +1,4 @@
+0.8.1.1
+-------
+
+- Fix tests compiling with servant-0.9

--- a/servant-mock.cabal
+++ b/servant-mock.cabal
@@ -1,5 +1,5 @@
 name:                servant-mock
-version:             0.8.1
+version:             0.8.1.1
 x-revision:          1
 synopsis:            Derive a mock server for free from your servant API types
 description:
@@ -14,7 +14,7 @@ maintainer:          haskell-servant-maintainers@googlegroups.com
 copyright:           2015-2016 Servant Contributors
 category:            Web
 build-type:          Simple
-extra-source-files:  include/*.h
+extra-source-files:  README.md CHANGELOG.md include/*.h
 cabal-version:       >=1.10
 bug-reports:         http://github.com/haskell-servant/servant-mock/issues
 

